### PR TITLE
Translated Render & Commit Page

### PR DESCRIPTION
**Pull request by Rohan-Makwana**
Updated the `Render & Commit` Page with Hindi Translations.
[render-and-commit.md](https://github.com/reactjs/hi.reactjs.org/files/7918509/render-and-commit.md)

### CODE REVIEW

File: beta/src/pages/learn/render-and-commit.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-* Why rendering does not always produce a DOM update
+* React में रेंडरिंग का क्याअर्थ है
+* React कब और क्यों एक कॉम्पोनेन्ट रेंडर करता है
+* स्क्रीन पर एक कॉम्पोनेन्ट प्रदर्शित करने में शामिल कदम
+* क्यों रेंडरिंग हमेशा एक डोम अद्यतन उत्पन्न नहीं करता है
 
 </YouWillLearn>
 
-Imagine that your components are cooks in the kitchen, assembling tasty dishes from ingredients. In this scenario, React is the waiter who puts in requests from customers and brings them their orders. This process of requesting and serving UI has three steps:
+कल्पना कीजिए कि आपके कंपोनेंट्स रसोई में रसोइया हैं, सामग्री से स्वादिष्ट व्यंजन इकट्ठा करते हैं। इस परिदृश्य में, React वेटर है जो ग्राहकों से अनुरोध करता है और उन्हें उनके आदेश लाता है। UI का अनुरोध करने और उसे प्रस्तुत करने की इस प्रक्रिया के तीन चरण हैं
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:20]: `के तीन चरण हैं` => `के तीन स्टेप्स हैं`

File: beta/src/pages/learn/render-and-commit.md [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
 
-1. **Triggering** a render (delivering the guest's order to the kitchen)
-2. **Rendering** the component (getting the order from the kitchen)
-3. **Committing** to the DOM (placing the order on the table)
+1. **ट्रिग्गरिंग** रेंडर (रसोईघर में अतिथि का आदेश पहुंचाना)
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:22]: `आदेश` => `आर्डर`

File: beta/src/pages/learn/render-and-commit.md [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
+2. **रेंडरिंग** कॉम्पोनेन्ट (रसोई से आदेश प्राप्त करना)
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:23]: `कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:23]: `आदेश प्राप्त करना` => `आर्डर लेना`

File: beta/src/pages/learn/render-and-commit.md [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
+3. **कमिटिंग** डोम पर (टेबल पर ऑर्डर देकर)
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:24]: `देकर` => `देना`

File: beta/src/pages/learn/render-and-commit.md [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
 
 <IllustrationBlock sequential>
-  <Illustration caption="Trigger" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
-  <Illustration caption="Render" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
-  <Illustration caption="Commit" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
+  <Illustration caption="ट्रिगर" alt="React as a server in a restaurant, fetching orders from the users and delivering them to the Component Kitchen." src="/images/docs/illustrations/i_render-and-commit1.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:27]: `ट्रिगर` => `Trigger`

File: beta/src/pages/learn/render-and-commit.md [hunk 6]
```diff
@@ -1,44 +1,44 @@
...
+  <Illustration caption="रेंडर" alt="The Card Chef gives React a fresh Card component." src="/images/docs/illustrations/i_render-and-commit2.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:28]: `रेंडर` => `Render`

File: beta/src/pages/learn/render-and-commit.md [hunk 7]
```diff
@@ -1,44 +1,44 @@
...
+  <Illustration caption="कमिट" alt="React delivers the Card to the user at their table." src="/images/docs/illustrations/i_render-and-commit3.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:29]: `कमिट` => `Commit`

File: beta/src/pages/learn/render-and-commit.md [hunk 8]
```diff
@@ -1,44 +1,44 @@
...
 </IllustrationBlock>
 
-## Step 1: Trigger a render {/*step-1-trigger-a-render*/}
+## स्टेप 1: एक रेंडर ट्रिगर करें {/*step-1-trigger-a-render*/}
 
-There are two reasons for a component to render:
+किसी कॉम्पोनेन्ट को रेंडर करने के दो कारण हैं:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:34]: `कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

File: beta/src/pages/learn/render-and-commit.md [hunk 9]
```diff
@@ -1,44 +1,44 @@
...
 
-1. It's the component's **initial render.**
-2. The component's **state has been updated.**
+1. यह कॉम्पोनेन्ट का **प्रारंभिक रेंडर है।**
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:36]: `कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

File: beta/src/pages/learn/render-and-commit.md [hunk 10]
```diff
@@ -1,44 +1,44 @@
...
+2. कॉम्पोनेन्ट की **स्थिति अपडेट कर दी गई है।**
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:37]: `कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

File: beta/src/pages/learn/render-and-commit.md [hunk 11]
```diff
@@ -1,44 +1,44 @@
...
 
-### Initial render {/*initial-render*/}
+### प्रारंभिक रेंडर {/*initial-render*/}
 
-When your app starts, you need to trigger the initial render. Frameworks and sandboxes sometimes hide this code, but it's done by calling `ReactDOM.render` with your root component and the target DOM node:
+जब आपका ऐप शुरू होता है, तो आपको शुरुआती रेंडर को ट्रिगर करना होगा। फ़्रेमवर्क और सैंडबॉक्स कभी-कभी इस कोड को छुपाते हैं, लेकिन यह आपके मूल कॉम्पोनेन्ट और टारगेट DOM नोड के साथ ReactDOM.render को कॉल करके किया जाता है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:41]: `मूल कॉम्पोनेन्ट` => `रुट कौम्पोनॅन्ट`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.

Added review till line 65.

Lets get these closed, the we can continue further with review.

For our review process you can refer #23

File: beta/src/pages/learn/conditional-rendering.md [hunk 12]
```diff
@@ -257,7 +257,7 @@ export default function PackingList() {
```
[the remark this answers is not in the document]
Comment (Rohan-Makwana) [reply beta/src/pages/learn/conditional-rendering.md:257]: No major change as such, just backspaces at a couple of places, will rectify it by adding the spaces back.

File: beta/src/pages/learn/render-and-commit.md (hunk 2 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:22]: ऑर्डर or आर्डर, I Think ऑर्डर should be the correct one. What do you say?

File: beta/src/pages/learn/render-and-commit.md (hunk 7 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:29]: Shall I also change the captions of the second illustration starting at line 75? to the English ones

**Comment by Rohan-Makwana** [comment]
@arshadkazmi42  Thank you, I have done the changes you have asked for till line 65, also  I made sure that whatever word changes or spelling changes you have suggested I have changed it globally throughout the file, awaiting your response for some doubts will commit once they are cleared.

File: beta/src/pages/learn/render-and-commit.md (hunk 7 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/render-and-commit.md:29]: @Rohan-Makwana 

Yes. anything which is part of code, we don't want to translate those

File: beta/src/pages/learn/conditional-rendering.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (arshadkazmi42) [reply beta/src/pages/learn/conditional-rendering.md:257]: Yes. I guess this should be fine

File: beta/src/pages/learn/render-and-commit.md (hunk 2 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/render-and-commit.md:22]: Yes. makes sense, can you update it in whole file.

Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:22]: @arshadkazmi42 yes I have updated it throughout the file, I have also updated all the other words you've requested throughout the file.

File: beta/src/pages/learn/render-and-commit.md (hunk 11 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:41]: Hey, @arshadkazmi42 one more question should performance be written as it is in Hindi or translate it to the Hindi meaning of it?

Comment (arshadkazmi42) [reply beta/src/pages/learn/render-and-commit.md:41]: Lets write it in hindi, without translation, it will make more sense

**Comment by Rohan-Makwana** [comment]
@arshadkazmi42 updated the `Render and Commit` page with all the changes you mentioned. I think the file is perfect now. Awaiting your review.

File: beta/src/pages/learn/render-and-commit.md [hunk 13]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
 
 </Sandpack>
 
-Try commenting out the `ReactDOM.render` call and see the component disappear!
+`ReactDOM.render` कॉल पर कमेंट करने का प्रयास करें और देखें कि कौम्पोनॅन्ट गायब हो गया है|
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:68]: `गया है|` => `गया है!`

File: beta/src/pages/learn/render-and-commit.md [hunk 14]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
 
-### Re-renders when state updates {/*re-renders-when-state-updates*/}
+### स्टेट अपडेट होने पर पुन: रेंडर {/*re-renders-when-state-updates*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:70]: `स्टेट` => `state`

I think i missed this. but we are not translating `state`

Can you find and replace this in whole file

File: beta/src/pages/learn/render-and-commit.md [hunk 15]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
 
-Once the component has been initially rendered, you can trigger further renders by updating its state with [`setState`](reference/setstate). Updating your component's state automatically queues a render. (You can imagine these as a restaurant guest ordering tea, dessert, and all sorts of things after putting in their first order, depending on the state of their thirst or hunger.)
+एक बार कौम्पोनॅन्ट शुरू में रेंडर किया गया है, तो आप [`setState`](reference/setstate) के साथ इसकी स्टेट को अपडेट करके आगे के रेंडरर्स को ट्रिगर कर सकते हैं|
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:72]: I think you missed this line to translate 

```
Updating your component's state automatically queues a render. (You can imagine these as a restaurant guest ordering tea, dessert, and all sorts of things after putting in their first order, depending on the state of their thirst or hunger.)
```

can you translate this also, seems this is part of line 72

File: beta/src/pages/learn/render-and-commit.md [hunk 16]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
 
 <IllustrationBlock sequential>
-  <Illustration caption="State update..." alt="React as a server in a restaurant, serving a Card UI to the user, represented as a patron with a cursor for their head. They patron expresses they want a pink card, not a black one!" src="/images/docs/illustrations/i_rerender1.png" />
-  <Illustration caption="...triggers..." alt="React returns to the Component Kitchen and tells the Card Chef they need a pink Card." src="/images/docs/illustrations/i_rerender2.png" />
-  <Illustration caption="...render!" alt="The Card Chef gives React the pink Card." src="/images/docs/illustrations/i_rerender3.png" />
+  <Illustration caption="State Update..." alt="React as a server in a restaurant, serving a Card UI to the user, represented as a patron with a cursor for their head. They patron expresses they want a pink card, not a black one!" src="/images/docs/illustrations/i_rerender1.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:75]: `Update` => `update`

File: beta/src/pages/learn/render-and-commit.md [hunk 17]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
+  <Illustration caption="...Triggers..." alt="React returns to the Component Kitchen and tells the Card Chef they need a pink Card." src="/images/docs/illustrations/i_rerender2.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:76]: `Triggers` => `triggers`

File: beta/src/pages/learn/render-and-commit.md [hunk 18]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
+  <Illustration caption="...Render!" alt="The Card Chef gives React the pink Card." src="/images/docs/illustrations/i_rerender3.png" />
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:77]: `Render` => `render`

File: beta/src/pages/learn/render-and-commit.md [hunk 19]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
 </IllustrationBlock>
 
-## Step 2: React renders your components {/*step-2-react-renders-your-components*/}
+## स्टेप 2: React आपके कौम्पोनॅन्टस को रेंडर करता है {/*step-2-react-renders-your-components*/}
 
-After you trigger a render, React calls your components to figure out what to display on screen. **"Rendering" is React calling your components.**
+एक रेंडर ट्रिगर करने के बाद, React आपके कौम्पोनॅन्टस को यह पता लगाने के लिए कॉल करता है कि स्क्रीन पर क्या डिस्प्ले किया जाए। **"रेंडरिंग" पर React आपके कौम्पोनॅन्टस को कॉल कर रहा है |**
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:82]: `**"रेंडरिंग" पर React आपके कौम्पोनॅन्टस को कॉल कर रहा है |**` => `**"रेंडरिंग" मतलब, React आपके कौम्पोनॅन्टस को कॉल कर रहा है।**`

File: beta/src/pages/learn/render-and-commit.md [hunk 20]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
 
-* **On initial render,** React will call the root component.
-* **For subsequent renders,** React will call the function component whose state update triggered the render.
+* **प्रारंभिक रेंडर पर**, React रूट कौम्पोनॅन्ट को कॉल करेगा।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:84]: `**प्रारंभिक रेंडर पर**,` => `**प्रारंभिक रेंडर पर,**`

File: beta/src/pages/learn/render-and-commit.md [hunk 21]
```diff
@@ -65,28 +65,28 @@ export default function Image() {
...
+* **बाद के रेंडर के लिए,** React उस फंक्शन कौम्पोनॅन्ट को कॉल करेगा जिसके स्टेट अपडेट ने रेंडर को ट्रिगर किया।
 
-This process is recursive: if the updated component returns some other component, React will render _that_ component next, and if that component also returns something, it will render _that_ component next, and so on. The process will continue until there are no more nested components and React knows exactly what should be displayed on screen.
+यह प्रोसेस पुनरावर्ती है: यदि अपडेट कौम्पोनॅन्ट किसी अन्य कौम्पोनॅन्ट को लौटाता है, तो React उस कौम्पोनॅन्ट को अगला रेंडर करेगा, और यदि वह कौम्पोनॅन्ट भी कुछ लौटाता है, तो वह उस कौम्पोनॅन्ट को आगे रेंडर करेगा, और इसी तरह। प्रोसेस तब तक जारी रहेगी जब तक कि कोई और नेस्टेड कौम्पोनॅन्ट न हों और React को ठीक से पता हो कि स्क्रीन पर क्या डिस्प्ले किया जाना चाहिए।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:87]: `वह उस कौम्पोनॅन्ट` => `वह _उस_ कौम्पोनॅन्ट`

File: beta/src/pages/learn/render-and-commit.md [hunk 22]
```diff
@@ -128,34 +128,35 @@ img { margin: 0 10px 10px 0; }
...
-* **Mind its own business.** It should not change any objects or variables that existed before rendering. (One order should not change anyone else's order.)
+* **समान इनपुट, वही आउटपुट।** समान इनपुट को देखते हुए, एक कौम्पोनॅन्ट को हमेशा वही JSX वापस करना चाहिए। (जब कोई टमाटर के साथ सलाद का ऑर्डर देता है, तो उसे प्याज के साथ सलाद नहीं मिलना चाहिए!)
+* **अपना खुद का व्यवसाय ध्यान रखें।** इसे रेंडर करने से पहले मौजूद किसी भी ऑब्जेक्ट्स या वेरिएबल्स को नहीं बदलना चाहिए।(एक ऑर्डर से किसी और का ऑर्डर नहीं बदलना चाहिए।)
 
-Otherwise, you can encounter confusing bugs and unpredictable behavior as your codebase grows in complexity. When developing in "Strict Mode," React calls each component's function twice, which can help surface mistakes caused by impure functions.
+अन्यथा, आप भ्रमित बग और अप्रत्याशित व्यवहार का सामना कर सकते हैं क्योंकि आपका कोडबेस जटिलता में बढ़ता है। "स्ट्रिक्ट मोड" में डेवेलोप होने पर, React प्रत्येक कौम्पोनॅन्ट के कार्य को दो बार कॉल करता है, जो अशुद्ध कार्यों के कारण सतह की गलतियों में मदद कर सकता है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:142]: `जटिलता` => `कम्प्लेक्सिटी`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:142]: `"स्ट्रिक्ट मोड"` => `"Strict Mode"`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:142]: `डेवेलोप` => `डेवेलप`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:142]: `अशुद्ध कार्यों` => `इमप्योर फंक्शन्स`

File: beta/src/pages/learn/render-and-commit.md [hunk 23]
```diff
@@ -128,34 +128,35 @@ img { margin: 0 10px 10px 0; }
...
 
 </Gotcha>
 
-<DeepDive title="Optimizing performance">
+<DeepDive title="ऑप्टिमिज़िंग परफॉरमेंस">
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:146]: `<DeepDive title="ऑप्टिमिज़िंग परफॉरमेंस">` => `<DeepDive title="Optimizing performance">`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Good going.

I have added review till line 146.

Lets get these closed then we can continue further.

File: beta/src/pages/learn/render-and-commit.md (hunk 14 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:70]: Replaced state in the entire file.

File: beta/src/pages/learn/render-and-commit.md (hunk 15 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:72]: Queue will remain as it is right?

Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:72]: I have translated the above line as below, any changes needed?
`अपने कौम्पोनॅन्ट की स्टेट को अपडेट करने पर ऑटोमेटिकली एक रेंडर केओए होता है। (आप उनकी प्यास या भूख की स्थिति के आधार पर, उनके पहले ऑर्डर में डालने के बाद चाय, मिठाई और सभी प्रकार की चीजों को ऑर्डर करने वाले एक रेस्तरां अतिथि के रूप में कल्पना कर सकते हैं।)`

Comment (arshadkazmi42) [reply beta/src/pages/learn/render-and-commit.md]: > केओए होता है।

`केओए होता है।` => `रेंडर कियु हो जाता है।`

File: beta/src/pages/learn/render-and-commit.md [hunk 24]
```diff
@@ -128,34 +128,34 @@ img { margin: 0 10px 10px 0; }
...
 
 <DeepDive title="Optimizing performance">
 
-The default behavior of rendering all components nested within the updated component is not optimal for performance if the updated component is very high in the tree. If you run into a performance issue, there are several opt-in ways to solve it described in the [Performance](/learn/performance) section. **Don't optimize prematurely!**
+अपडेट कौम्पोनॅन्ट के भीतर नेस्टेड सभी कौम्पोनॅन्टस को रेंडर करने का डिफ़ॉल्ट व्यवहार परफॉरमेंस के लिए ऑप्टिमम नहीं है यदि अपडेट कौम्पोनॅन्ट ट्री में बहुत ऊंचाई पर है। यदि आप किसी परफॉरमेंस समस्या का सामना करते हैं, तो इसे हल करने के लिए [परफॉरमेंस](/learn/performance) अनुभाग में वर्णित कई ऑप्ट-इन तरीके हैं। **समय से पहले अनुकूलन न करें!**
 
 </DeepDive>
 
-## Step 3: React commits changes to the DOM {/*step-3-react-commits-changes-to-the-dom*/}
+## स्टेप 3: React DOM में कमिटस करता है {/*step-3-react-commits-changes-to-the-dom*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:151]: `React DOM में कमिटस करता है` => `React DOM में चेंजेस कमिट करता है`

File: beta/src/pages/learn/render-and-commit.md [hunk 25]
```diff
@@ -128,34 +128,34 @@ img { margin: 0 10px 10px 0; }
...
 
-After rendering (calling) your components, React will modify the DOM. 
+आपके कौम्पोनॅन्टस को रेंडर करने (कॉल करने) के बाद, React DOM में संशोधित करेगा।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:153]: `संशोधित` => `मॉडिफाई`

let me know if you have any alternate for this

File: beta/src/pages/learn/render-and-commit.md [hunk 26]
```diff
@@ -128,34 +128,34 @@ img { margin: 0 10px 10px 0; }
...
 
-* **For the initial render,** React will use the [`appendChild()`](https://developer.mozilla.org/docs/Web/API/Node/appendChild) DOM API to put all the DOM nodes it has created on screen. 
-* **For re-renders,** React will apply the minimal necessary operations (calculated while rendering!) to make the DOM match the latest rendering output.
+* **प्रारंभिक रेंडर पर,** React स्क्रीन पर बनाए गए सभी DOM नोड्स को डालने के लिए [`appendChild ()`](https://developer.mozilla.org/docs/Web/API/Node/appendChild)  DOM API का उपयोग करेगा।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:155]:
```
[`appendChild ()`]
```
=>
```
[`appendChild()`]
```

File: beta/src/pages/learn/render-and-commit.md [hunk 27]
```diff
@@ -128,34 +128,34 @@ img { margin: 0 10px 10px 0; }
...
+* **पुन: रेंडर करने के लिए,** DOM को नवीनतम रेंडरिंग आउटपुट से मिलान करने के लिए React न्यूनतम आवश्यक संचालन (रेंडरिंग के दौरान परिकलित!) लागू करेगा।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:156]:
```
DOM को नवीनतम रेंडरिंग आउटपुट से मिलान करने के लिए React न्यूनतम आवश्यक संचालन (रेंडरिंग के दौरान परिकलित!) लागू करेगा।
```
=>
```
React DOM को लेटेस्ट आउटपुट के जैसा करने के लिए कम से कम ज़रूरी ऑपरेशन्स (जो रेंडरिंग करते वक़्त कैलकुलेट होते है!) का इस्तेमाल करेगा।    
```

File: beta/src/pages/learn/render-and-commit.md [hunk 28]
```diff
@@ -128,34 +128,34 @@ img { margin: 0 10px 10px 0; }
...
 
-**React only changes the DOM nodes if there's a difference between renders.** For example, here is a component that re-renders with different props passed from its parent every second. Notice how you can add some text into the `<input>`, updating its `value`, but the text doesn't disappear when the component re-renders:
+**React केवल DOM नोड्स को बदलता है यदि रेंडरर्स के बीच कोई अंतर है। **उदाहरण के लिए, यहां एक कौम्पोनॅन्ट है जो हर सेकेंड में अपने पैरेंट से पारित विभिन्न प्रोप के साथ फिर से रेंडर करता है। ध्यान दें कि आप `<input>` में कुछ टेक्स्ट कैसे जोड़ सकते हैं, इसके `value` को अपडेट कर सकते हैं, लेकिन जब कौम्पोनॅन्ट फिर से रेंडर होता है तो टेक्स्ट गायब नहीं होता है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:158]:
```
**React केवल DOM नोड्स को बदलता है यदि रेंडरर्स के बीच कोई अंतर है। **उदाहरण
```
=>
```
**React केवल DOM नोड्स को बदलता है यदि रेंडरर्स के बीच कोई अंतर है।** उदाहरण
```

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:158]: `पारित` => `पास किये गए`

File: beta/src/pages/learn/render-and-commit.md [hunk 29]
```diff
@@ -195,21 +195,21 @@ export default function App() {
 
 </Sandpack>
 
-This works because during this last step, React only updates the content of `<h1>` with the new `time`. It sees that the `<input>` appears in the JSX in the same place as last time, so React doesn't touch the `<input>`—or its `value`!
-## Epilogue: Browser paint {/*epilogue-browser-paint*/}
+यह काम करता है क्योंकि इस अंतिम स्टेप के दौरान, React केवल `<h1>` की कंटेंट को नए `time` के साथ अपडेट करता है। यह देखता है कि `<input>` JSX में पिछली बार की तरह ही दिखाई दे रहा है, इसलिए React `<input>`— या इसके `value` को स्पर्श नहीं करता है!
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:198]: `स्पर्श नहीं करता` => `नहीं छूता`

File: beta/src/pages/learn/render-and-commit.md [hunk 30]
```diff
@@ -195,21 +195,21 @@ export default function App() {
...
+## उपसंहार: ब्राउज़र पेंट {/*epilogue-browser-paint*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:199]: `उपसंहार` => `Epilogue`

looks too complex. lets keep it as is

File: beta/src/pages/learn/render-and-commit.md [hunk 31]
```diff
@@ -195,21 +195,21 @@ export default function App() {
...
 
-After rendering is done and React updated the DOM, the browser will repaint the screen. Although this process is known as "browser rendering", we'll refer to it as "painting" to avoid confusion in the rest of these docs.
+रेंडरिंग के बाद और React ने DOM को अपडेट करने पर, ब्राउज़र स्क्रीन को फिर से रंग देगा। हालांकि इस प्रोसेस को "ब्राउज़र रेंडरिंग" के रूप में जाना जाता है, हम इन बाकी डाक्यूमेंट्स में भ्रम से बचने के लिए इसे "पेंटिंग" के रूप में संदर्भित करेंगे।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:201]: `React ने DOM` => `React के DOM`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:201]: `भ्रम` => `कन्फूज़न`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:201]: `"पेंटिंग"` => `"painting"`

Comment (arshadkazmi42) [inline beta/src/pages/learn/render-and-commit.md:201]: `संदर्भित` => `रेफेर`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work with the fixes.

I have added review for rest of the file. 

Once these are fixed, we can get this merged.

File: beta/src/pages/learn/render-and-commit.md (hunk 25 above)
Comment (Rohan-Makwana) [reply beta/src/pages/learn/render-and-commit.md:153]: सुधार ??

**Comment by Rohan-Makwana** [comment]
@arshadkazmi42 Done with the changes you mentioned. 
I think this pull request #179 is ready to be merged, awaiting your review. 
Thank you.

File: beta/src/pages/learn/render-and-commit.md (hunk 25 above)
Comment (arshadkazmi42) [reply beta/src/pages/learn/render-and-commit.md]: Lets keep `मॉडिफाई` only

**Review by arshadkazmi42 — APPROVED** [review]
Great work :tada: 
Let's get this merged.

Thank you for working on this

**Pull request merged**